Jibauni (#2)
Alrodel/endless-sky: data/jibauni/jibauni 0 kanlanmo transport.txt

# Copyright (c) 2024 by Loymdayddaud
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Jibauni: Kanlanmo 0"
	name "Take Kanlanmo to <planet>"
	description "Kanlanmo, who is a Jibauni in the shape of a Tyrannosaurus rex, has asked to go to <planet> on your ship. He will explain once you get there."
	landing
	source Baitoon
	destination Taltahard
	on offer
		conversation
			branch jibauniinsider
				has "species: Jibauni"

> 			`One of the weird aliens who appear to dwell here and who seem to be in the shape of the mythic Tyrannosaurus Rex, approaches your ship as you land. When you land and start out of your ship, grabbing your sidearm and donning armor as you go, it is standing by the hatch. "Hello, fair stranger," it says. "How may I serve you on this delightful day?"`

Author: Updated. How does it look now?

			choice
				`	"I came through the wormhole. Do you mean to say you have not noticed it?"`
					goto wormholeentry
				`	"I came with a Jump Drive."`
					to display
						has "outfit (flagship installed): Jump Drive"
					goto jumpdriveentry
				`	"What do you mean, purple?"`
					goto purple
			label purple
			`	"Oh, it does not matter. A side note, I believe. But, to rephrase my question, how did you enter our space?"`
			choice
				`	"This anomaly called a wormhole."`
					goto wormholeentry
				`	"Just my fancy Jump Drive."`
					goto jumpdriveentry
			label jumpdriveentry
			`	"Interesting," says the alien. "The only one who has ever done that before is an odd ship who seemed to come from the same general direction." It fixes you with an intent gaze. "There may be something similar between your species. The ship was pink. Seen one like it?"`
				goto pinkship
			label rudeentry
			`	"There is an odd ship who seemed to come the way you did," says the alien as it fixes you with an intent gaze. "There may be something similar between your species. The ship was pink. Seen one like it?"`
				goto pinkship
			label pinkship
			choice

> 				`	"Yes, I have, actually, called the Dominator, an Alpha ship."`

Author: My guess is that most players will meet, or know of, the Alphas long before anybody finds the Rulei. Maybe fifteen percent find the Rulei, but anybody who has done FW knows of the Alphas.

> 				`	"Yes, I have, actually, called the Dominator, an Alpha ship."`

Author: The one thing about that is the Jibauni never actually saw the Alphas face to face. They just know their ship was pink.

			choice
				`	"Okay. I'll take you."`
					accept
				`	"That's what you think. Well, I'm joining the Rebels, whatever you may say."`
					goto joinedrebels
				`	"No, I won't take you. Sorry."`
					decline
			label joinedrebels
			action
				"reputation: Jibauni" = -10
				"reputation: Jibauni (Civilian)" = -10
				"reputation: Jibauni (Rebel)" = 10
				set "jibauni: joined the rebels"
			`	"Very well. Goodbye, and never land here again until you have made reparations."`
				flee
			label jibauniinsider
			action
				event "jibauni translated hails"
				set "language: Jibauni"
			`A Jibauni comes up to you when you walk out of the shipyard. "Could you take me to <planet> for <payment>? It is something to do with the Rebel Jibauni, and I need to have a meeting with someone. My name is Kanlanmo.`
			choice
				`	"If you would like."`
					accept
				`	"No, sorry. I can't right now."`
					decline
	on complete
		payment 15000
		set "First Contact: Jibauni: offered"
		conversation
			branch insider
				has "species: Jibauni"
			action
				event "jibauni translated hails"
				set "language: Jibauni"

> 			`"Thank you for bringing me here so promptly," says Kanlanmo. "It was very kind of you. Now, I know you are burning with questions, but I have an urgent meeting. If you meet me in the spaceport soon, though, I can answer your questions. Also, here is a translator you can use, so you can talk to other Jibauni." He goes off after handing you <payment>, and you sit back to wait until the time he specified.`

Author: It's not meant to be explained well - he's being mysterious, not knowing how much he can trust you.

# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Jibauni: Kanlanmo 0"
	name "Take Kanlanmo to <planet>"
	description "Kanlanmo, a Jibauni in the shape of a Tyrannosaurus Rex, has asked to go to <planet> on your ship. He will explain more once you get there."
	landing
	source Baitoon
	destination Taltahard
	on offer
		conversation
			branch jibauniinsider
				has "species: Jibauni"
			`Your landing location is on a large set of old landing pads. There didn't seem to be anywhere else that was available for spaceships to arrive. The city is filled with dinosaurs in the shape of the mythical Tyrannosaurus Rex, although here they are smaller then the old fables. A while after you land, one of these dinosaurs approaches your ship.`
			choice
				`	(Go out and meet it.)`
					goto meeting
				`	(Leave now, before something bad happens.)`
					depart
				`	(Stay in the ship.)`

> 			`	The dinosaur opens its mouth, and a moment later you hear the words, "Could you come out of your ship to have a meeting?"`

Author: ```suggestion
			`	The dinosaur opens its mouth, and a moment later a growling sound rumbles through the air. Then you hear the words, "Could you come out of your ship to have a meeting?"`
```

				to display
					not "language: Coalition"
				goto initialchoice1
			action
				set "knows the Jibauni have Heliarch translators"

> 			`	The dinosaur opens its mouth to say something, but you notice that it has a Coalition translator box around its neck. Before you can do anything it says, "Could you come out of your ship to have a meeting?"`

Author: ```suggestion
			`	The dinosaur opens its mouth to say something, and you hear a growling sound. Then you notice that it has a Coalition translator box around its neck. Before you can do anything the box says, "Could you come out of your ship to have a meeting?"`
```

				to display
					has "language: Coalition"
			label initialchoice1
			choice
				`	(Go outside the <ship> and meet the strange beast.)`
					goto meeting
				`	(Leave the planet.)`
					defer
			label meeting
			`	"Hello, fair stranger," it says. "Is there a way I may serve you on this delightful day?"`
			label meetingquestions
			choice
				`	"Hello. My name is <first> <last>, and I come in peace."`
					goto lastisknown
				`	"How do you have a Coalition translator?"`
					to display
						has "knows the Jibauni have Heliarch translators"
						not "asked about the Jibauni translators"
					goto cotrans
				`	"Hi, weirdo. What's up?"`
					goto rude
				`	(Shoot the alien with your sidearm.)`
					goto deathshot
			label cotrans
			action
				set "asked about the Jibauni translators"
			`	"I am not authorized to give you that information. So how may I serve you?"`
				goto meetingquestions
			label lastisknown

> 			`	"Yes. We know about you, <first>, Captain of the <ship>. How have you come here, for we did not detect any purple sparks?"`

Author: They don't know that the wormhole can be landed on.
They've managed to pick up slight words from inside your ship, and one of them was repeated so many times they felt sure it was the captain's name.

				goto pinkship
			label alphas
			`	You begin the story, telling everything you know about how the Alphas tried to enslave human space.`
			`	At the end, he says, "Wow. That is a long story. And I take it you are not on their side?"`
			choice
				`	"No, of course not."`
					goto alphasoffer
				`	"Yes, I have been friendly with them for years."`
					goto friendlyalpha
			label alphasoffer
			`	"Good. Well, that just leaves one more matter to be wrapped up. Can you take me to <planet> in the <system> system? Kanlanmo of the Jibauni will be grateful."`
			choice
				`	"Of course!"`
					accept
				`	"Why?"`
					goto whychoice
				`	"Not yet, I have something I need to do first. But I'll be back shortly!"`
					decline
			label rude
			`	The alien announces, though to what you cannot see, "The stranger has been ignominiously impolite." It turns to you. "What do you say, 'weirdo'? Will you apologize? Or not, and see what happens?"`
			choice
				`	"I apologize, and did not mean to insult you."`
					goto rudeentry
				`	"Why apologize to a bunch of dinky T-Rex's?"`
					goto deathexplode
			label deathexplode

> 			`	A cluster of anti-ship weaponry is fired from all sides, immediately exploding the <ship> and everything inside it. It is sad your life had to end in such a way.`

Author: ```suggestion
			`	A cluster of anti-ship weaponry is fired from all sides, immediately exploding the <ship> and everything inside it. It is sad your life had to end in such a way.`
			`	The moment the last word is out of your mouth, the dinosaur lowers its hand. Suddenly, millions of miniature projectiles come hurtling through the air, shredding the <ship> and everything in it. It is sad that your life had to end this way.`
```

				die
			label deathshot

> 			`	You pull your sidearm from its holster, but before you can fire it, hidden anti-ship weaponry trains fire on your ship. Caught in crossfire, the <ship> immediately explodes, blowing you and everything in it sky high.`

Author: ```suggestion
			`	You pull your sidearm from its holster, but before you can fire it, hidden anti-ship weaponry trains fire on your ship. Caught in crossfire, the <ship> immediately explodes, blowing you and everything in it sky high.`
			`	The moment your hand touches your sidearm, a whining noise is heard. A huge missile comes down with a shriek, followed by another and another. <ship> is obliterated in a moment, leaving you in a pile of smoldering ash. You are taken to jail and imprisoned for life.`
```

				die
			label friendlyalpha
			action
				"reputation: Jibauni" = -1000
				"reputation: Jibauni (Civilian)" = -1000
				"reputation: Jibauni (Fueler)" = -1000

> 				"reputation: Jibauni (Rebel)" = -1000

Author: Because, while the Jibauni and Rebel Jibauni are on fighting terms, they loathe the Alphas infinitely more than they could ever feel emotion for each other.

The Jibauni, hearing what First Last has to say, are going to send a truce ship to give the message.

			`	"Really? In that case, we must ask you to leave our planet immediately and never come back. Goodbye, and don't let us see your face ever again!"`
				flee
			label whychoice
			action
				set "jibauni: first contact: why"
			`	"Because I have some business to transact, something about the Rebel Jibauni."`
			choice
				`	"Who are the Rebel Jibauni?"`
					goto rebeljibauni
				`	"Sounds good. I'll take you."`
					accept
				`	"No. If it has to do with another faction, I can't."`
					decline
			label rebeljibauni
			`	"They are, as the name might suggest, Jibauni who have split away from the real government. They are not very polite, and you definitely do not want to be on their side."`
			choice
				`	"Okay. I'll take you."`
					accept
				`	"That's what you think. Well, I'm joining the Rebels, whatever you may say."`
					goto joinedrebels
				`	"No, I won't take you. Sorry."`
					decline
			label joinedrebels

> 			action

Author: Because they'll notice you coming out of Jibauni space with hostile ships trailing you.

If they haven't received an Alpha message, they'll assume you were rude to the Jibauni... and anybody who does that I surely a friend of the Rebels.

				"reputation: Jibauni" = -10
				"reputation: Jibauni (Civilian)" = -10
				"reputation: Jibauni (Rebel)" = 10
				set "jibauni: joined the rebels"
			`	"Very well. Goodbye, and never land here again until you have made reparations."`
				flee
			label jibauniinsider
			action
				event "jibauni translated hails"
				set "language: Jibauni"
			`A Jibauni comes up to you when you walk out of the shipyard. "Could you take me to <planet> for <payment>? It is something to do with the Rebel Jibauni, and I need to have a meeting with someone. My name is Kanlanmo.`
			choice
				`	"If you would like."`
					accept
				`	"No, sorry. I can't right now."`
					decline
	on complete
		payment 15000
		set "First Contact: Jibauni: offered"
		conversation
			branch insider
				has "species: Jibauni"
			action
				event "jibauni translated hails"
				set "language: Jibauni"

> 			`"Thank you for bringing me here so promptly," says Kanlanmo. "It was very kind of you. Now, I know you are burning with questions, but I have an urgent meeting. If you meet me in the spaceport soon, though, I can answer your questions. Also, here is a translator you can use, so you can talk to other Jibauni." He goes off after handing you <payment>, and you sit back to wait until the time he specified.`

Author: ```suggestion
			`"Thank you for bringing me here so promptly," says Kanlanmo. "It was very kind of you. Now, I know you are burning with questions, but I have an urgent meeting. If you meet me in the spaceport soon, though, I can answer your questions. Also, here is a translator you can use, so you can talk to other Jibauni." He goes off after handing you <payment>, and you sit back to wait until the time he specified.`
				to display
					not "language: Coalition"
			`"Thank you for bringing me here so promptly," says Kanlanmo. "It was very kind of you. Now, I know you are burning with questions, but I have an urgent meeting. If you meet me in the spaceport soon, though, I can answer your questions. Here is a translator you can use for ease of communication with other Jibauni. Yes, I am aware you have a box of the same make, but this is slightly different." He goes off after handing you <payment>, and you sit back to wait until the time he specified.`
				to display
					has "language: Coalition"
```

			choice
				`	"No, of course not."`
					goto alphasoffer
				`	"Yes, I have been friendly with them for years."`
					goto friendlyalpha
			label alphasoffer
			`	"Good. Well, that just leaves one more matter to be wrapped up. Can you take me to <planet> in the <system> system? Kanlanmo of the Jibauni will be grateful."`
			choice
				`	"Of course!"`
					accept
				`	"Why?"`
					goto whychoice
				`	"Not yet, I have something I need to do first. But I'll be back shortly!"`
					decline
			label rude
			`	The alien announces, though to what you cannot see, "The stranger has been ignominiously impolite." It turns to you. "What do you say, 'weirdo'? Will you apologize? Or not, and see what happens?"`
			choice
				`	"I apologize, and did not mean to insult you."`
					goto rudeentry
				`	"Why apologize to a bunch of dinky T-Rex's?"`
					goto deathexplode
			label deathexplode
			`	The moment the last word is out of your mouth, the dinosaur lowers its hand. Suddenly, millions of miniature projectiles come hurtling through the air, shredding the <ship> and everything in it. It is sad that your life had to end this way.`
				die
			label deathshot

> 			`	The moment your hand touches your sidearm, a whining noise is heard. A huge missile comes down with a shriek, followed by another and another. <ship> is obliterated in a moment, leaving you in a pile of smoldering ash. You are taken to jail and imprisoned for life.`

Reviewer: Just reminding the player that First Last isn't in the ship, and slight tweak as First Last would certainly be dead from the explosion if it left _them_ surrounded by a pile of ash.
```suggestion
			`	The moment your hand touches your sidearm, a whining noise is heard. A huge missile comes down with a shriek, followed by another and another. The <ship> behind you is obliterated in a moment, leaving it a pile of smoldering ash. You are taken to jail and imprisoned for life.`
```
